tests/std/tests/P0896R4_views_split/test.cpp

```diff
@@ -292,13 +253,23 @@ struct instantiator {
             Read empty{span<const char, 0>{}};
             test_one(move_if_needed(empty), views::empty<char>, views::empty<string_view>);
         }
-        if constexpr (ranges::forward_range<Read>) { // Range delimiter
+        { // Range delimiter
             Read read{span{text}};
             test_one(move_if_needed(read), "is"sv, expected_range);
 
             Read empty{span<const char, 0>{}};
             test_one(move_if_needed(empty), "is"sv, views::empty<string_view>);
         }
+
+        { // LWG-3478, trailing empty range
+            Read read{span{trailing_empty}};
+            test_one(move_if_needed(read), ' ', expected_trailing);
+        }
+
+        { // LWG-3505, mutlichar pattern

```

Comment:
I congratulate you on your consistent spelling of "mutlichar" =)
```suggestion
        { // LWG-3505, multichar pattern
```